packages/platform-ui/cypress/pages/auth/utils/getNewUser.ts

```diff
@@ -0,0 +1,18 @@
+import {ValidUser} from "../../common/interfaces/Auth";
+import faker from 'faker'
+
+export const getFakeEmail = () => {
+  return `${faker.internet.email()}.test`
+};
+
+export const getNewUser = (): ValidUser => {
+  const email = getFakeEmail();
+  return {
+    user: {
+      email: email,
```

Comment:
```suggestion
      email,
```
Prefer the short syntax

Reply:
done , thanks